--- utils/grib_repack	2016-01-18 20:33:14.000000000 -0300
+++ utils/grib_repack	2016-04-18 08:40:27.465564215 -0300
@@ -65,7 +65,7 @@
         sys.stdout.write('not a GRIB2 message, skipping ..\n')
         continue
     if grb['dataRepresentationTemplateNumber'] == ipack:
-	sys.stdout.write('no repacking required, skipping ..\n')
+        sys.stdout.write('no repacking required, skipping ..\n')
     grb['dataRepresentationTemplateNumber'] = ipack
     msg = grb.tostring()
     f.write(msg)
